.github/workflows/code-checks.yml

```diff
@@ -43,7 +43,8 @@ jobs:
         env:
           FREEZE_REQUIREMENTS: 1
         run: |
-          pip install -e '.[all]' -r requirements/typing.txt
+          # TODO: investigate hanging installation with app sub-package
+          pip install -e '.[pytorch,fabric,data]' -r requirements/typing.txt
```

Comment:
Should this be?

```suggestion
          pip install -e '.[pytorch-all,fabric-all,data-all]' -r requirements/typing.txt
```

Reply:
This doesn't work, can we undo it?

Reply:
Thanks. I'll work backwards from this in #18698 to resolve!